blog/content/post/2022-03-07-meta-nezha.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Introduction of Yocto meta layer for Nezha D1
+abstract: 'Presentation of current progress status with the support of the
+Nezha board in Yocto Project'
+cover: /img/nezha-logo.png
+author: cezary.sobczak
+layout: post
+published: true
+date: 2022-03-07
+archives: "2022"
+
+tags:
+  - risc-v
+  - sbc
+  - linux
+  - u-boot
+  - boot0
+  - opensbi
+  - bootloader
+  - logs
+  - SoC
+  - sdcard
+  - yocto
+  - layer
+  - meta
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Introduction
+
+As it was mentioned in first [post](https://blog.3mdeb.com/2021/2021-11-19-nezha-riscv-sbc-first-impression/)
+We are interested in support for this board in Yocto Project, and this post will
+show you what we have achieved in this field.
+
+Previous blog post made a lot of noise and was appreciated by people from RISC-V
+International. Now it is also available on their [site](https://riscv.org/news/2022/01/first-impression-on-nezha-risc-v-sbc-3mdeb/).
+
+Whole code of the meta layer you can find at [github](https://github.com/Cezarus27/meta-nezha).
+It should be mentioned that `meta-nezha` is really early support so
+the recipe's code isn't that clear and for now, it isn't prepared to be
+upstreamed.
+
+## Nezha D1 meta layer
+
+### What is a Yocto meta layer?
```

Comment:
@Cezarus27 not sure if this should be new subsection + leaving empty start of section does not look good. I would suggest to remove this `### What is a Yocto meta layer?` and edit the next to paragraphs to slightly explain what the Yocto Project is and why creating a layer for Nezha D1 is a nice thing to do